### ISSUE TITLE
Using elapsed on a string field in a subquery sends blank values

### ISSUE THREAD

**Issue by Hyruu** [influxdata/influxdb#9124]
__InfluxDB version:__ 1.3.6
__Bug:__ Using the ELAPSED function inside a subquery while specifying a field containing string values sends back a column 'elapsed' containing only blank values. Using the ELAPSED function on a field containing numeric values works as intended.

__Example:__
The *visits* table contains a numeric field *duration*, a string field *device* and a tag *zone*.
```
> SELECT elapsed("device",1d) FROM "visits" GROUP BY "zone"
name: visits
tags: zone=Zone_1
time                elapsed
----                -------
1434440322000000000 1
1434440603000000000 4
1434440734000000000 2

name: visits
tags: zone=Zone_2
time                elapsed
----                -------
1434440636000000000 3
1434440655000000000 0
```
```
> SELECT "elapsed" FROM (SELECT elapsed("device",1d) FROM "visits" GROUP BY "zone")
name: visits
time                elapsed
----                -------
1434440322000000000
1434440603000000000
1434440734000000000
1434440636000000000
1434440655000000000
```

Yet, the following works:
```
> SELECT elapsed("duration",1d) FROM "visits" GROUP BY "zone"
name: visits
tags: zone=Zone_1
time                elapsed
----                -------
1434440322000000000 1
1434440603000000000 4
1434440734000000000 2

name: visits
tags: zone=Zone_2
time                elapsed
----                -------
1434440636000000000 3
1434440655000000000 0
```
```
> SELECT "elapsed" FROM (SELECT elapsed("duration",1d) FROM "visits" GROUP BY "zone")
name: visits
time                elapsed
----                -------
1434440322000000000 1
1434440603000000000 4
1434440734000000000 2
1434440636000000000 3
1434440655000000000 0
```

The problem seems to be exclusive to the ELAPSED function, as other functions such as COUNT worked properly in both cases.

**Comment by jsternberg** [comment]
I'm going to check the cause, but this appears to be fixed in v1.4. I'm not sure why yet.

**Comment by jsternberg** [comment]
It looks like this is fixed in 1.4 from this commit: https://github.com/influxdata/influxdb/commit/0ef94e0cf06f0a776e713ec3cb462df62dcbe9b2#diff-187b262c5908de7e030346b970b398c3

If you notice the part that mentions `elapsed`, that's the code that fixes it.
